cmd/ig/utils/flags.go

```diff
@@ -60,6 +60,8 @@ type CommonFlags struct {
 
 func AddCommonFlags(command *cobra.Command, commonFlags *CommonFlags) {
 	command.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
+		command.Parent().PersistentPreRunE(command.Parent(), args)
+
```

Comment:
Why do we need to do this?

Reply:
> When the parent command and child command both have PersistentPreRunE, the child command's function would be called instead, so bindViper was not being called with list-containers command.

We could call the parent's `PersistentPreRunE` explicitly when sub-command is overriding it, but this wasn't showing results for env var set value, so instead using `PreRunE`.